modules/app/variables.tf

```diff
@@ -29,9 +29,16 @@ variable "domain" {
   type        = string
 }
 
-variable "image" {
+variable "images" {
   description = "The Octo STS application image."
-  default     = "chainguard/octo-sts:latest"
+  type = object({
+    app     = optional(string, "chainguard/octo-sts:latest")
+    webhook = optional(string, "chainguard/octo-sts-webhook:latest")
```

Comment:
This doesn't exist yet (obviously), but we'll need to create it once we cut the next release!